examples/sshd.yml

```diff
@@ -28,6 +28,10 @@ files:
     source: ~/.ssh/id_rsa.pub
     mode: "0600"
     optional: true
+  - path: root/.ssh/authorized_keys
```

Comment:
Does having two entries with identical `path` really DTRT in all possible combinations of `source` being available?

Perhaps we should have a special entry (akin to the `metadata` key) which specifies a list of public keys to harvest and construct an authorized_keys from? e.g. `authorized_keys: ["rsa", "ed25519"]` (or full paths, etc)